src/hotspot/share/gc/shenandoah/shenandoahUtils.hpp

```diff
@@ -47,16 +47,16 @@ class ShenandoahGeneration;
 #define SHENANDOAH_RETURN_EVENT_MESSAGE(generation_type, prefix, postfix) \
   switch (generation_type) {                                              \
     case NON_GEN:                                                         \
-      return prefix " (NON-GENERATIONAL)" postfix;                        \
+      return prefix " " postfix;                                          \
```

Comment:
I think this causes a double space, for example here between "roots" and "(unload classes)" here:

```
[2.736s][info][gc] GC(42) Concurrent weak roots  (unload classes) 0.740ms
```

Reply:
Yes, fixed here: https://github.com/openjdk/shenandoah/pull/540/commits/1c7752c121853c131a352025f3d87cc179549506